gogoproto/gogo.proto

```diff
@@ -38,6 +38,10 @@ extend google.protobuf.EnumOptions {
 	optional bool enum_stringer = 62022;
 }
 
+extend google.protobuf.EnumValueOptions {
+	optional string enum_customname = 66001;
```

Comment:
65xxx seems to be in use for field options: https://github.com/gogo/protobuf/blob/master/gogoproto/gogo.proto#L102.